src/main/java/jalau/at18/battlecity/Input.java

```diff
@@ -0,0 +1,21 @@
+package jalau.at18.battlecity;
+
+import java.util.Scanner;
+
+public class Input {
+    private Scanner scan;
+    private TankPlayer tankOne;
+
+    public Input() {
+        this.scan = new Scanner(System.in);
+        this.tankOne = new TankPlayer();
```

Comment:
Would it be better if the tank class calls the input class so that the game classes call the tank class instead of the input class?